src/entity/CommonUser.java

```diff
@@ -4,6 +4,7 @@
 
 class CommonUser implements User {
 
+  private final String email;
   private final String name;
```

Comment:
Are you sure we should have a name field instead of a unique username field? Make sure to consult @EquilibriaW on these changes so it doesn't effect the security/functionality of search PRs.

Reply:
Creating issue number 23 to track this issue